test-plan.md

```diff
@@ -128,11 +142,30 @@ The Blazor experience is available when you open a .cshtml file in a valid OmniS
 
 ##### C# Diagnostics
 * When no changes have been performed on `Pages/Index.cshtml`, there are 0 errors.
-* Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file. 
+* Typing `@ThisDoesNotExist` results in an error being created and squiggled in the .cshtml file.
+
+##### Components
+* Typing `<Counter>` prompts you with HTML completion for the `Counter` tag and on completion commit auto-completes the closing tag.
```

Comment:
You should also see the completion for the fully qualified Counter tag. Do we want to call that out?

Reply:
Ya I can add that.